src/components/LoginPage.tsx

```
@@ -1,110 +1,64 @@
-                  </button>
-                )}
-                <button type="submit" className="btn btn-primary flex-1">
-                  Entrar
-                </button>
-              </div>
-            </form>
+          <div className="flex flex-col gap-3">
+            <button
+              onClick={handleGitHubLogin}
+              className="btn btn-primary btn-lg gap-2"
+            >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)